docs/index.html

```diff
@@ -58,7 +59,16 @@ <h2>Agents</h2>
                   target="_blank"
                   class="metric-link"
                 >
-                  <span id="copilot-total">{{COPILOT_TOTAL}}</span>
+                  <span id="copilot-total">1,100</span>
```

Comment:
Don't hard code this - keep it as `COPILOT_TOTAL`.  Same for all the others in this file.